fastlane/.env.default

```diff
@@ -1,25 +1,29 @@
-## Fastlane Build - Project related variables
-CRU_FASTLANE_USERNAME="cruapps@cru.org"
-CRU_APP_IDENTIFIER="org.cru.godtools"
-CRU_TARGET="godtools"
-CRU_SCHEME="godtools"
-CRU_XCODEPROJ="godtools.xcodeproj"
-CRU_XCWORKSPACE="godtools.xcworkspace"
-CRU_TEST_SCHEME="godtoolsUITests"
-CRU_TEST_DEVICES="iPhone 7"
 
-CRU_DEFAULT_BRANCH="master"
 
-## AppStore release related variables
-CRU_API_KEY_PATH="./fastlane/AppleAppStoreAPI.json"
-CRU_IDFA_IS_ENABLED=true
-CRU_IDFA_TRACKS_INSTALL=true
+APPLE_DEVELOPER_TEAM_ID = "DQ48D9BF2V"
+APPLE_ID_USERNAME = "cruapps@cru.org"
```

Comment:
as far as I know, this option is no longer required now that we are using an API key